lib/Message/Passing/Filter/Encoder/Sereal.pm

```diff
@@ -0,0 +1,92 @@
+package Message::Passing::Filter::Encoder::Sereal;
+use Moo;
+use MooX::Types::MooseLike::Base qw( Bool HasMethods );
```

Comment:
On 2015-11-03 03:03 PM, Damien Krotkine wrote:
> In lib/Message/Passing/Filter/Encoder/Sereal.pm
> <https://github.com/suretec/Message-Passing/pull/10#discussion_r43799598>:
> 
>> @@ -0,0 +1,92 @@
>> +package Message::Passing::Filter::Encoder::Sereal;
>> +use Moo;
>> +use MooX::Types::MooseLike::Base qw( Bool HasMethods );
> 
> Why do you need MooX::Types::MooseLike::Base here ? Seems like you never
> use these types ?

Good point. I think it was a direct port (and lack of after-the-fact
cleanup) of the JSON serializer.  Pushed a commit that removes that line.